staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -190,6 +190,19 @@ func (s *GenericAPIServer) PrepareRun() preparedGenericAPIServer {
 // Run spawns the http servers (secure and insecure). It only returns if stopCh is closed
 // or one of the ports cannot be listened on initially.
 func (s preparedGenericAPIServer) Run(stopCh <-chan struct{}) error {
```

Comment:
I'm assuming it's no longer necessary to document the error case now that cleanup is performed automatically in that case.